microsoft-edge/devtools-guide-chromium/autofill/index.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Autofill: Inspect and debug saved addresses"
+description: Inspect and debug address info that's been saved to Microsoft Edge.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: devtools
+ms.date: 02/28/2025
+---
+<!-- Copyright Sofia Emelianova
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       https://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.  -->
+# Autofill: Inspect and debug saved addresses
+<!-- https://developer.chrome.com/docs/devtools/autofill -->
+
+Use the **Autofill** tool to inspect and debug address info that's been saved to Microsoft Edge.
+
+The **Autofill** tool records various events that the browser triggers when a user is filling in a form by using the browser's autofill feature -- for example, when the browser remembers your address and autofills it in an address field.
+
+
+<!-- ====================================================================== -->
+## Overview
+
+The **Autofill** tool provides a convenient way to automatically fill forms on websites with saved addresses.  The **Autofill** tool in DevTools lets you inspect the mapping between your form fields, predicted autofill values, and saved data.
+
+The **Autofill** tool lets you inspect only the address info.
+
+See also:
+* [Autofill](https://web.dev/learn/forms/autofill) at web.dev.
+
+
+<!-- ====================================================================== -->
+## Save address info to Edge
+<!-- https://developer.chrome.com/docs/devtools/autofill#save-to-chrome -->
+
+By default, Microsoft Edge prompts you to save address info you entered into a web form when you submit it.
+
+<!-- todo: find in Edge Settings -->
+
+<!-- ![A prompt to save address info](todo: autofill-prompt.png) -->
+
+<!-- If there's no such prompt, in Edge, select Customize and control > Password and Autofill > Addresses and more, and turn on Save and fill addresses. You can also add new addresses here. -->
```

Comment:
Does Edge support Autofill?
What's the Edge equiv of **Settings and more** > **Password and Autofill**?
@captainbrosset

Rendered article for review:
* **Autofill: Inspect and debug saved addresses**
   * `/devtools-guide-chromium/autofill/index.md`
   * Internal preview: https://review.learn.microsoft.com/microsoft-edge/devtools-guide-chromium/autofill/index?branch=pr-en-us-3392
   * External preview: https://github.com/MicrosoftDocs/edge-developer/blob/user/mikehoffms/autofill/microsoft-edge/devtools-guide-chromium/autofill/index.md
   * Upstream: https://developer.chrome.com/docs/devtools/autofill